src/javascripts/jquery.tipsy.js

```diff
@@ -70,30 +73,42 @@
                 }
 
                 var pos = $.extend({}, this.$element.offset(), {
-                    width: this.$element[0].getBoundingClientRect().width,
-                    height: this.$element[0].getBoundingClientRect().height
+                    width: this.$element[0].offsetWidth,
+                    height: this.$element[0].offsetHeight
```

Comment:
This overwrites the changes from PR #4 , could you update this to take that into account?